api/app/db/database.py

```diff
@@ -22,7 +22,8 @@
 # connect to database - defaulting to always use utc timezone
 _write_engine = create_engine(DB_WRITE_STRING, connect_args={
                               'options': '-c timezone=utc'})
-_read_engine = create_engine(DB_READ_STRING, connect_args={
+# use pre-pring on read, as connections are quite often stale due to how few users we have at the moment.
```

Comment:
pre-pring -> pre-ping if you feel necessary

Reply:
Thanks for catching my bad spelling - fixed.